craft_cli/errors.py

```diff
@@ -57,3 +57,18 @@ def __init__(
         self.docs_url = docs_url
         self.reportable = reportable
         self.retcode = retcode
+
+    def __eq__(self, other):
+        if isinstance(other, CraftError):
+            return all(
+                [
+                    self.args == getattr(other, "args"),
```

Comment:
Why not doing `other.args` directly?

Reply:
Good catch. This is a leftover before I did the `isinstance` check -> fixed